.github/workflows/checks.yml

```diff
@@ -72,6 +73,8 @@ jobs:
             os-version: windows-2019
           - python-version: "3.8"  # avoid uploading coverage for full matrix
             use_coverage: true
+          - python-version: "3.8"  # this is to avoid installing optional dependencies in all environments
```

Comment:
Does this mean that these tests will not work or that users will not automatically install TensorFlow when installing FOQUS in Python versions other than 3.8?

Reply:
This won't affect users, and is only a directive to specify that, in the environment where CI jobs run, `tensorflow` should only be installed for a subset of the combinations. Currently, there are 3 Python versions x 3 OSes = 9 total combinations, but the GUI tests only work for 3.7 and 3.8, so specifying `tensorflow` only for Python 3.8 is a way to test both "with TF" and "without TF" using jobs with Python 3.8 and 3.7 respectively.